src/util/path.js

```diff
@@ -729,6 +741,62 @@
     return result;
   };
 
+  /**
+   *
+   * Converts points to a smooth SVG path
+   * @param {Array} points Array of points
+   * @param {number} [correction] Apply a correction to the path (usually we use `width / 1000`). If undefined it will be inferred using `points`.
+   * @return {(string|number)[][]} An array of SVG path commands
+   */
+  function getSmoothPathFromPoints(points, correction) {
+    var path = [], i,
+        p1 = new fabric.Point(points[0].x, points[0].y),
+        p2 = new fabric.Point(points[1].x, points[1].y),
+        len = points.length, multSignX = 1, multSignY = 0, manyPoints = len > 2;
+
+    //  if no correction is passed we infer it by calculating the path's width
+    if (correction === undefined) {
+      var start = points[0].x;
+      var minMax = points.reduce(
+        function (prev, curr) {
+          return {
+            min: Math.min(curr.x, prev.min),
+            max: Math.max(curr.x, prev.max)
+          };
+        },
+        { min: start, max: start }
+      );
+      var width = minMax.max - minMax.min;
+      correction = width / 1000;
+    }
```

Comment:
if correction is undefined, we set it to 0. we don't want to correct every path. is just a edge case of the brush.
please remove this if, let's just add a `correction = correction || 0;` and this PR is good to go

Reply:
500abd6, af9045e
e740a16 extracts the logic to a separate util. What do you think?